clang/test/CIR/CodeGen/neon-tmp.c

```diff
@@ -0,0 +1,228 @@
+// RUN: %clang_cc1 -triple arm64-none-linux-gnu -target-feature +neon \
+// RUN:    -fclangir -disable-O0-optnone \
+// RUN:  -flax-vector-conversions=none -emit-cir -o %t.cir %s
+// RUN: FileCheck --check-prefix=CIR --input-file=%t.cir %s
```

Comment:
This file can probably be merged into `neon.c` or get a more meaningful name.

Reply:
Ah, that's why I have the PR as draft, I'll never check in this file. Just use it to make sure added test cases are right before adding them into existing test file. A long test file is harder to debug/change :-)

Reply:
yep, but since I was already here I left a comment just in case hehe!

Reply:
test cases went to neon.c